numpy/_typing/_add_docstring.py

```diff
@@ -120,7 +120,7 @@ def _parse_docstrings() -> str:
 
 add_newdoc('NDArray', repr(NDArray),
     """
-    A `np.ndarray[Any, np.dtype[+ScalarType]] <numpy.ndarray>` type alias 
+    A `np.ndarray[Any, np.dtype[+ScalarType]] <numpy.ndarray>` type alias
     :term:`generic <generic type>` w.r.t. its `dtype.type <numpy.dtype.type>`.
 
     Can be used during runtime for typing arrays with a given dtype
```

Comment:
I'd append a sentence here with something along the line of "See `npt.NDArray <numpy.typing.NDArray>` for dealing with arrays of a specified shape."; something to easily allow one to find the `npt.Array` docs starting from `npt.NDArray`.